<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Social Media</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="social_lbl">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SOCIAL MEDIA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="facebook_pb">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>320</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Facebook</string>
    </property>
   </widget>
   <widget class="QPushButton" name="instagram_pb">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>320</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Instagram</string>
    </property>
   </widget>
   <widget class="QPushButton" name="whatsapp_pb">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>320</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Whatsapp</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>941</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
